src/latte_kernel/nbe.cljc

```diff
@@ -144,7 +145,7 @@
     ;; We call it now with the appropriate argument to extract the body.
     (stx/binder? t)
     (let [[binder [x tx] f] t
-          x' (symbol (str "_" level))
+          x' (with-meta (symbol (str "_" level)) {::name x})
```

Comment:
I think the `with-meta` here should only be effected if there is not already a `::name` key available

Reply:
Fixed, but doesn't change the problem